rfcs/text/0030/process.yml

```diff
@@ -127,3 +127,26 @@
         Entry point information for a session.
 
         Remote client information such as ip, port and geo location.
+
+    - name: tty
```

Comment:
process.tty is needed in v1 of session viewer, so it has been changed to a reduced set from file_description (which actually works better in the implementation)

Reply:
Yes, this is cleaner - the file descriptor information was unnecessary so re-using the FileDescription struct was confusing.  A TTY may be attached to several open FDs (often stdin, stdout & stderr).